marketplace_confg/chart/dicom-adapter/templates/manifest.yaml

```diff
@@ -0,0 +1,128 @@
+            - "--store_compress_to_transfer_syntax={{ .Values.import.store_compress_to_transfer_syntax }}"
+            {{- end }}
+            {{- if .Values.import.monitoring_project_id }}
+            - "--monitoring_project_id={{ .Values.import.monitoring_project_id }}"
+          env:
+            - name: ENV_POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: ENV_POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: ENV_CONTAINER_NAME
+              value: dicom-import-adapter
+            {{- end }}
+        {{- if .Values.export.hasexporter }}
+        - name: dicom-export-adapter
+          image: "{{ .Values.export.image.repo }}:{{ .Values.export.image.tag }}"
+          args:
+            {{- if .Values.export.peer_dimse_aet }}
+            - "--peer_dimse_aet={{ .Values.export.peer_dimse_aet }}"
+            {{- end }}
+            {{- if .Values.export.peer_dimse_ip }}
+            - "--peer_dimse_ip={{ .Values.export.peer_dimse_ip }}"
+            {{- end }}
+            {{- if .Values.export.peer_dimse_port }}
+            - "--peer_dimse_port={{ .Values.export.peer_dimse_port }}"
+            {{- end }}
+            {{- if .Values.export.peer_dicomweb_address }}
+            - "--peer_dicomweb_address={{ .Values.export.peer_dicomweb_address }}"
+            {{- end }}
+            {{- if .Values.export.project_id }}
+            - "--project_id={{ .Values.export.project_id }}"
+            {{- end }}
+            {{- if .Values.export.subscription_id }}
+            - "--subscription_id={{ .Values.export.subscription_id }}"
+            {{- end }}
+            {{- if .Values.export.dicomweb_addr }}
+            - "--dicomweb_addr={{ .Values.export.dicomweb_addr }}"
+            {{- end }}
+            {{- if .Values.export.oauth_scopes }}
+            - "--oauth_scopes={{ .Values.export.oauth_scopes }}"
+            {{- end }}
+            {{- if .Values.export.use_gcp_application_default_credentials }}
+            - "--use_gcp_application_default_credentials"
+            {{- end }}
+            {{- if .Values.export.monitoring_project_id }}
+            - "--monitoring_project_id={{ .Values.export.monitoring_project_id }}"
+            env:
+            - name: ENV_POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: ENV_POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: ENV_CONTAINER_NAME
+              value: dicom-export-adapter
+            {{- end }}
+        {{- end }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-dicom-adapter-lb
+  {{- if .Values.lb.internal }}
+  annotations:
+    cloud.google.com/load-balancer-type: "Internal"
```

Comment:
Would this deployment work without the Internal annotation? Do we know of a use case that doesn't use Internal? if not might be fine to leave this as always internal

Reply:
Yes it will work without the internal annotation, without the internal flag users can access the adapter using a external IP.
I am not sure if there exists such a use case and I mainly expose the lb for test purposes. For example, I can use dcmsend on my computer to send dcm files to the import adapter.

Reply:
Can we make lb.internal default to true, to match our readme recommendation and keep it more locked down by default?

Reply:
That makes sense. Fixed.